bigdata/src/main/scala/org/w3/banana/bigdata/BigdataShared.scala

```diff
@@ -0,0 +1,32 @@
+package org.w3.banana.bigdata
+
+import com.bigdata.rdf.model._
+import org.w3.banana.{RDFOpsModule, RDFModule, RDF}
+
+/**
+ * Shared classes between Bigdata Remote and Local
+ */
+trait BigdataShared
+{
```

Comment:
I will delete BigdataShared as I only need banana-rdf support for embeded bigdata and will not have time for implementing remote one in the nearest months. If somebody will get interested in remote or I will change my mind then we will thing what to do with types, as the difference between embeded and remote modes in bigdata is types of queries.